test/src/xquery/fn.xql

```diff
@@ -21,4 +41,17 @@ declare
 function fnt:normalize-unicode($normalization-form as xs:string) {
     count(string-to-codepoints(normalize-unicode($fnt:composed, $normalization-form))),
     count(string-to-codepoints(normalize-unicode($fnt:decomposed, $normalization-form)))
-};
\ No newline at end of file
+};
+
+declare
```

Comment:
@shabanovd That would introduce a dependency on an external website into the test suite. The problem is that if that website is offline our test suite fails. So I don't want an external dependency.